Updating info and originals for openage

simonsan opened the pull request:
Merge after #757

pehsa commented:
No problem here, just the SW: GB not in the originals yet. After the  #757 merge, the Travis build will pass all tests

simonsan commented:
I just found out about the Video-Tag and also added our status update and added some more infos, after talking to the team. Still talking about the change of content type. Sorry for the inconvenience. :-(
